.circleci/config.yml

```diff
@@ -20,11 +20,11 @@ jobs:
           command: yarn run build-all
       - run:
           name: Run the site server
-          command: PORT=80 ./bin/serve
+          command: PORT=8016 ./bin/serve
```

Comment:
Maybe this should instead be `yarn run build-all` followed-by `yarn run start-server` to be more production-like. Then at least it hopefully would have caught this error before @anthonydillon tried to release it, if it happened again.